sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -2566,9 +2579,11 @@ public void multiLabelClassification(HttpClient httpClient, TextAnalyticsService
         client = getTextAnalyticsAsyncClient(httpClient, serviceVersion);
         classifyCustomMultiLabelRunner((documents, parameters) -> {
             SyncPoller<ClassifyDocumentOperationDetail, ClassifyDocumentPagedFlux> syncPoller =
-                client.beginMultiLabelClassify(documents, parameters.get(0), parameters.get(1), "en", null).getSyncPoller();
+                client.beginMultiLabelClassify(documents, parameters.get(0), parameters.get(1), "en",
+                    new MultiLabelClassifyOptions().setDisplayName("operationName")).getSyncPoller();
             syncPoller = setPollInterval(syncPoller);
-            syncPoller.waitForCompletion();
+            PollResponse<ClassifyDocumentOperationDetail> pollResponse = syncPoller.waitForCompletion();
+            assertEquals("operationName", pollResponse.getValue().getDisplayName());
```

Comment:
nitpick: The test case can be more robust if the "operationName" value comes directly from the options bag object, so if for some reason it changed, there is no need to change the assert line too.